src/services/sync/strategies/TSP/TSPSchoolSyncer.js

```diff
@@ -417,8 +421,8 @@ class TSPSchoolSyncer extends mix(Syncer).with(ClassImporter) {
 			this.stats.errors.push({
 				type: 'create-student',
 				entity: tspStudent.schuelerUid,
-				message: `Schüler "${tspStudent.schuelerVorname} ${tspStudent.schuelerNachname}"`
-					+ ' konnte nicht erstellt werden.',
+				message:
+					`Schüler "${tspStudent.schuelerVorname} ${tspStudent.schuelerNachname}"` + ' konnte nicht erstellt werden.',
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)